src/magicbox/SuggestionsManager.ts

```diff
@@ -138,6 +138,7 @@ export class SuggestionsManager {
       // By definition, once an element has been "selected" with the keyboard,
       // it is not longer "active" since the event has been processed.
       this.keyboardFocusedElement = null;
+      this.inputManager.blur();
```

Comment:
Without triggering a `click` event, the suggestions menu stays open after pressing `enter` on a preview. Blurring the omnibox prevents this new behaviour.

Reply:
Alright, we will also need a unit test checking that `blur` is called.

Reply:
Alright, I'll add a test for that.